addons/event/i18n/event.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.1alpha1+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-04 09:24+0000\n"
-"PO-Revision-Date: 2022-10-04 09:24+0000\n"
+"POT-Creation-Date: 2022-11-14 12:56+0000\n"
```

Comment:
Don't update meta data, we keep the original data (updates are part of file history anyway).